.github/workflows/ci.yml

```diff
@@ -318,7 +318,7 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - name: Cargo Run E2E Tests (Fuel VM)
         run: |
-          fuel-core run --db-type in-memory --debug & 
+          fuel-core run --db-type in-memory --debug --snapshot ./.github/workflows/local-testnode & 
```

Comment:
Why does it need the snapshot?

Reply:
Fuel-Core removed their default state which we were depending on for our test suite. Basically to sign deployment transactions for e2e tests that requires deployment